src/components/select/select.js

```diff
@@ -288,19 +288,16 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $compile, $par
             }
           });
 
-        // Wait until postDigest so that we attach after ngModel's
-        // blur listener so we can set untouched.
-        $mdUtil.nextTick(function () {
-          element.on('blur', function() {
-            if (untouched) {
-              untouched = false;
-              ngModelCtrl.$setUntouched();
-            }
 
-            if (selectScope.isOpen) return;
-            containerCtrl && containerCtrl.setFocused(false);
-            inputCheckValue();
-          });
+        element.on('blur', function() {
```

Comment:
Yes, if you're going to remove the whole line: "scope.$evalAsync(ngModelCtrl.$setUntouched);", then yes, it's going to fail. What I meant, is that the whole change done to those lines are not needed for the fix based on tabindex to work. Furthermore, as I said, those changes are not completely correct, because they are missing the "isOpen" condition, which is already included in https://github.com/angular/material/pull/7646. If you'are going to change those lines to the previous state:
```
        // Wait until postDigest so that we attach after ngModel's
        // blur listener so we can set untouched.
        $mdUtil.nextTick(function () {
          element.on('blur', function() {
            if (untouched) {
              untouched = false;
              ngModelCtrl.$setUntouched();
            }

            if (selectScope.isOpen) return;
            containerCtrl && containerCtrl.setFocused(false);
            inputCheckValue();
          });
        });
```
then no test is going to fail.